src/components/Display.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <div id="display">
+    <div className="row">
+      <p className="result">{result}</p>
+    </div>
+  </div>
```

Comment:
### [Optional]
we could reduce the creation of extra element, that was the main reason why you're using fragment in `app.js`

I think you could simplify this file like this: 👇 
```
return (
    <div>
      <p>{result}</p>
    </div>
  );
```

Reply:
Thanks, I will implement that change.